src/lib/metricsMgr/MetricsManager.cpp

```diff
@@ -142,8 +142,26 @@ void MetricsManager::add(const std::string& srv, const std::string& subServ, con
     subService = &subService[1];
   }
 
+  //
+  // Convert tenant/service to lowercase
```

Comment:
This seems to be a rather generic string processing... Don't we have some functional already for this (probably in the commons/ library)? In negative case, shouldn't we move this to a generic function?